extensions/amp-story/0.1/amp-story-bookend.css

```diff
@@ -0,0 +1,273 @@
+/**
```

Comment:
Just a general question that applies to all of these: do we still need `!important` everywhere since they're in the shadow root?

Reply:
Unfortunately yes, for the browsers that don't support shadow DOM, one could write CSS to override our rules...